index.html

```diff
@@ -87,7 +96,115 @@ <h5>Multi-Post Stories</h5>
                                     <button type="button">See project</button>
                                 </div>
                             </article>
-                            <!-- cards -->
+
+                            <!-- DESKTOP cards -->
+
+                            <div class="desktop_projects">
+
+                                  <article id="card1">
+                                    <!-- <img class="card1Img" src="Images/Mask Group.png" alt="card1image"> -->
+                                        <img class="cardImg" src="images/desktop_img_Placeholder_1.png" alt="card1img">
+                                       <!-- <h5>Profesional Art Printing Data</h5>
+                                            <p>
+                                              A daily selection of privately personalized reads; no accounts or
+                                              sign-ups required. Has been the industry's standard.
+                                            </p>
+                                            <ul>
+                                              <li class="list1">HTML</li>
+                                              <li class="list2">Bootstrap</li>
+                                              <li class="list3">Ruby</li>
+                                            </ul> -->
+                                            <button type="button" class="card1-button">See project</button>
+                                  </article>
+
+                                  <article id="card2">
+                                    <!-- <img class="card1Img" src="Images/Mask Group.png" alt="card1image"> -->
+                                        <img class="cardImg" src="images/desktop_img_Placeholder_2.png" alt="card2img">
+                                        <h5>Data Dashboard<br>Healthcare</h5>
+                                            <p>
+                                              A daily selection of privately personalized reads; no accounts or
+                                              sign-ups required. Has been the industry's standard.
+                                            </p>
+                                            <ul>
+                                              <li class="list1">HTML</li>
+                                              <li class="list2">Bootstrap</li>
+                                              <li class="list3">Ruby</li>
+                                            </ul>
+                                    <!--    <button type="button" class="card2-button">See project</button> -->
+                                  </article>
```

Comment:
- [ ] Kindly adjust the margins, and the button of the cards to match the figma design.

- Your design
![image](https://user-images.githubusercontent.com/41163571/230712365-65929a0b-edd7-4ecf-a716-ea805fd645dc.png)

Reply:
Done.